lib/client.js

```diff
@@ -149,8 +149,9 @@ var GiantSwarm = (function () {
      * @param {String} [uri] API route to be called, starting with "/"
      * @param {Function} [successCallback] Callback to be called in case of success, gets response data as argument
      * @param {Function} [errorCallback] Optional callback to be called in case of an error, error object as argument
+     * @param {Object} [params] URL parameters to be added (optional)
      */
-    _simple_data_get_request: function(uri, successCallback, errorCallback) {
+    _simple_data_get_request: function(uri, successCallback, errorCallback, params) {
```

Comment:
params should be the second argument. Callbacks are normally always the last one.